raddb/policy.d/canonicalization

```diff
@@ -111,3 +111,16 @@ rewrite_calling_station_id {
                 noop
         }
 }
+
+set_calling_station_id {
+        if (&User-Name && (&User-Name =~ /^${policy.mac-addr-regexp}$/i)) {
```

Comment:
Yes sure, btw this is only use in accounting.
OK so let's start the name challenge, we have to find a username with a or b or c or d or e or f with 12 characters long !